### PR TITLE
Add Liquality Wallet

### PR DESCRIPTION
**Pull request by jessgusclark**
Update rLogin to support add liquality wallet and updates the `eventListeners` to `on`. 

The checklist below shows the functions that worked. Surprisingly migrating with addresses, which uses two transactions, worked, while setting a subdomain's owner did not work.

The change to the listeners was also tested with Metamsk, Nifty, and WalletConnect.

### CODE REVIEW

**Comment by jessgusclark** [comment]
# RNS Manager Integration Checklist Template

- [ ] Search and Resolve
- [ ] Register
- [ ] Register refresh page (localstorage)

---
## Admin

- DomainInfo Screen
    - [ ] Set Controller
    - [ ] Reclaim
    - [ ] Transfer

- Your Addresses
    - [ ] Set New Address
    - [ ] Edit existing Address
    - [ ] Delete Address
    - [ ] Migrate Resolver + Addresses

- Subdomains
    - [ ] Create Subdomain
    - [ ] Edit Subdomain
    - [ ] Delete Subdomain

- Resolver 
    - [ ] Set Resolver
    - [ ] Set Content Bytes Record
- Reverse
    - [ ] Set Reverse

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "eth-ens-namehash": "^2.0.8",
     "history": "^4.7.2",
     "js-sha3": "^0.8.0",
+    "lodash": "^4.17.14",
```
Comment (ilanolkies) [inline package.json:20]: Why is `lodash` added?

Comment (jessgusclark) [reply package.json:20]: hahaha, I dunno, that was from your commit! I'll remove it!

Comment (ilanolkies) [reply package.json]: Jaja right... fixing it

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**